docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,112 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* TBD Spooling protocol is stable. Link to docs! ({issue}`24214`)
```

Comment:
fyi @wendigo .. will link to docs once I got that PR updated and merged with your help... and also reword this entry